### DISCUSSION TITLE
Hello World! Say Hi here!

### DISCUSSION THREAD

**Discussion by TANSixu** [SUSTech-CS305-Fall22/CS305-Project-Skeleton#4] [Show and tell]
Why not say hi to our TA team under this post?

**Comment by mingt2019** [comment]
Hi ALL~ We have very nice TAs~
Should you have any questions, you are welcome to post them here in discussions~~

**Reply by TANSixu** [reply]
Hi Prof!

**Reply by XiaoMingZHM** [reply]
Hi!

**Reply by CutieDeng** [reply]
Hi.

**Reply by Godblessmycode1** [reply]
Hi

**Reply by TANSixu** [reply]
> Hi

Hi godblessmycode!
You are the first student who says hi to us. So we decide to give you a............. sugar! 🍬!!!

**Reply by JoeyZi1** [reply]
Hi～

**Reply by TANSixu** [reply]
> Hi～

Hi Joey! Good to see you here!!!

**Reply by ChristinaLJC** [reply]
Hi~

**Reply by TANSixu** [reply]
> Hi~

Hi Christina!

**Reply by Lynchrocket** [reply]
Hi~

**Reply by TANSixu** [reply]
> Hi~

Hi Lynchrocket!

**Reply by wyatt-feng** [reply]
Hi TANSixu! You are really nice and helpful indeed:-)

**Reply by TANSixu** [reply]
> Hi TANSixu! You are really nice and helpful indeed:-)

Hi dazhi! Thanks for saying that, it means a lot to me! You all did a great job in this project. Looking forward to your excellent presentation tomorrow! Again, really appreciate it!
